src/config/dynamic-theme-fixes.config

```diff
@@ -3244,6 +3244,23 @@ INVERT
 
 ================================
 
+churchofjesuschrist.org
+
+INVERT
+#mainLogo
+.mainLogo
```

Comment:
I don't see any element matching this selector.

Reply:
![churchmainlogoid](https://user-images.githubusercontent.com/38667455/174845600-8fff1223-6035-40b9-bd64-9a1ed0791e78.png)
![churchmainlogoclass](https://user-images.githubusercontent.com/38667455/174845965-f2e30157-3ec9-4fdf-983f-5b36d451edd1.png)